docs/source/common/lightning_module.rst

```diff
@@ -1070,23 +1074,41 @@ recurrent network trajectories."
 
     class MyModel(LightningModule):
 
-        def __init__(self):
+        def __init__(self, input_size, hidden_size, num_layers, output_size,
+                     truncated_bptt_steps, criterion):
             super().__init__()
+            self.lstm = nn.LSTM(
+                input_size=input_size,
+                hidden_size=hidden_size,
+                num_layers=num_layers,
+                batch_first=True,
+            )
+            self.linear = nn.Linear(hidden_size, outpur_size)
+            self.criterion = criterion
+
             # Important: This property activates truncated backpropagation through time
             # Setting this value to 2 splits the batch into sequences of size 2
-            self.truncated_bptt_steps = 2
+            self.truncated_bptt_steps = truncated_bptt_steps
 
         # Truncated back-propagation through time
         def training_step(self, batch, batch_idx, hiddens):
+            x, y = batch
+
             # the training step must be updated to accept a ``hiddens`` argument
             # hiddens are the hiddens from the previous truncated backprop step
-            out, (hiddens, _) = self.lstm(data, hiddens)
+            out, hiddens = self.lstm(x, hiddens)
+
+            y_hat = self.linear(out[:, -1])
+            loss = self.criterion(y_hat, y)
```

Comment:
the details of the training step were left out intentionally, this doesn't have to be a runnable example btw, you can just put ellipses.

Reply:
Sure, I will remove them.

Btw, I do feel like there should eb a runable experiement somewhere. When I got started with lightning I looked everywhere on the web and could not find one.